src/main/java/uk/gov/digital/ho/hocs/casework/client/auditclient/AuditClient.java

```diff
@@ -617,5 +620,35 @@ public void addAppealOfficerInfo(AuditPayload.AppealItem.AppealItemBuilder build
         }
     }
 
+    public void createSuspensionAudit(ActionDataSuspension suspensionEntity) {
+
+        AuditPayload.ActionAuditPayload payload = new AuditPayload.SuspensionItem(
+                suspensionEntity.getUuid(),
+                suspensionEntity.getCaseTypeActionUuid(),
+                suspensionEntity.getActionSubtype(),
+                suspensionEntity.getCaseTypeActionLabel(),
+                suspensionEntity.getCaseDataType(),
+                suspensionEntity.getCaseDataUuid(),
+                suspensionEntity.getDateSuspensionApplied(),
+                suspensionEntity.getDateSuspensionRemoved(),
+                EventType.CASE_SUSPENSION_APPLIED
+        );
+        saveActionAudit(payload);
+    }
+
+    public void updateSuspensionAudit(ActionDataSuspension suspensionEntity) {
 
+        AuditPayload.ActionAuditPayload payload = new AuditPayload.SuspensionItem(
+                suspensionEntity.getUuid(),
+                suspensionEntity.getCaseTypeActionUuid(),
+                suspensionEntity.getActionSubtype(),
+                suspensionEntity.getCaseTypeActionLabel(),
+                suspensionEntity.getCaseDataType(),
+                suspensionEntity.getCaseDataUuid(),
+                suspensionEntity.getDateSuspensionApplied(),
+                suspensionEntity.getDateSuspensionRemoved(),
+                EventType.CASE_UNSUSPENDED
+        );
+        saveActionAudit(payload);
+    }
```

Comment:
These are identical aside from the EventType, could combine them and have a second parameter to flip between applying suspension or unsuspending

Reply:
I've condensed the building of the payload to a separate method to reduce code duplication, however I've avoided creating a single public audit method, so as to fit the other patterns used in this client.